compiler/src/dotty/tools/dotc/reporting/ErrorMessageID.scala

```diff
@@ -215,6 +215,7 @@ enum ErrorMessageID(val isActive: Boolean = true) extends java.lang.Enum[ErrorMe
   case TailrecNestedCallID //errorNumber: 199
   case FinalLocalDefID // errorNumber: 200
   case NonNamedArgumentInJavaAnnotationID // errorNumber: 201
+  case MatchCaseNonNullableWildcardWarningID // errorNumber: 202
```

Comment:
Why do we need another error?

Reply:
I misunderstood the nature of the issue, this has been reverted